cmd/worker/funcs_kmod_tarball.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/worker"
+	"github.com/spf13/cobra"
+)
+
+var kmodTarballCmd = &cobra.Command{
+	Use:               "kmod-tarball",
+	Short:             "Manage kernel modules using tarball",
+	PersistentPreRunE: kmodTarballFuncPreRunE,
+}
```

Comment:
Instead of this new command group, how about `worker kmod load-tarball` or `worker kmod load --tarball`?
I think `kmod` is still a good umbrella group for the new loading-from-tarball command.

Reply:
there are a couple of arguments for using new command group:
1. we can initialize worker in the kmod and kmod-tarball functions, instead of initializing them with ifs in each of the Load/Unload commands
2. it gives us an alost complete separation between kmod and kmod-tarball code , which is great in terms of backporting.  Right now we have exactly one line that differs between upstream and downstream code.